addon/components/polaris-breadcrumbs.hbs

```diff
@@ -0,0 +1,41 @@
+<nav
```

Comment:
if there is no `breadcrumb`, we shouldn't be rendering anything. Can we do that here like [original version](https://github.com/Shopify/polaris-react/blob/master/src/components/Breadcrumbs/Breadcrumbs.tsx#L27) pls